### DISCUSSION TITLE
Sidekiq with superfetch - some jobs end up in dead queue with no trace

### DISCUSSION THREAD

**Discussion by marcdumontier** [sidekiq/sidekiq#6501] [General]
Hi

Is there any functionality that would cause a job to end up in dead queue with no stack trace (Error column in sidekiq ui). It's possible this job is killing the pod due to memory, but trying to understand how it ends up in the dead queue. It does not appear to have got into retry or exhausted them

**Comment by mperham** [comment]
Could it be a poison pill? See the super fetch wiki page.

**Reply by marcdumontier** [reply]
thanks Nike, that seems right
